### ISSUE TITLE
#-1177596970------------- Alternative text of images should not be repeated as text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1861]
<strong>PageName:</strong> FeedViewDetailsPage  

<strong>Source:</strong>`<img id="itemThumbnail" class="thumbnail-img js-thumbnail" src="https://velocityautomate.mapsdevext.arcgis.com/sharing/rest/content/items/d4f81ae9d48741f795eeeafb200bfdfe/info/thumbnail/ago_downloaded.png?token=9RK8DY0qvaqYcPFrrqGyoYFDXDccEfRwGocAgWn1h2QHvqS27CYPWeoGfDDhiWbouvrpPtFqOCpMftXaKvLAbf1OV5jkMmHXEJBTesSLghq9tSzoMXGc9D0wRPJkZ3Bfs0oNcIILWCqiHqISsWrMD4ztNJzOD8jS-U443iJ29IoLBq872gG1Rwl9i43E9z-OSkRXziomCH0njOseI5n-6_DkiXBS8AoZxFQGekihrsp8DgjOOLfkJ48UfuVTT3DS" alt="Open in Map Viewer">`

<strong>Element Location: </strong>#itemThumbnail

<strong>Impact: </strong>minor

<strong>Failure Summary:</strong> Fix all of the following:
  Element contains <img> element with alt text that duplicates existing text

<strong>Tags:</strong> cat.text-alternatives,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/image-redundant-alt?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedViewDetailsPage\-1177596970_Alternative text of images should not be repeated as text.png`
